api/src/main/kotlin/org/jellyfin/apiclient/api/client/KtorClient.kt

```diff
@@ -37,25 +48,32 @@ open class KtorClient(
 		}
 		.joinToString("/")
 
-
 	fun createAuthorizationHeader(): String? {
-		// TODO retrieve this via the library module (future PR)
 		val params = mutableMapOf(
-			"Client" to "CLIENTNAME",
-			"Device" to "DEVICENAME",
-			"DeviceId" to "DEVICEID",
-			"Version" to "VERSION"
+			"client" to clientInfo.name,
+			"version" to clientInfo.version,
+			"deviceId" to deviceInfo.id,
+			"device" to deviceInfo.name
 		)
 
-//		params["Token"] = ""
+		// Only set access token when it's not null
+		accessToken?.let { params["token"] = it }
 
-		// Format: `MediaBrowser key="value", key="value"`
+		// Format: `MediaBrowser key1="value1", key="value"`
```

Comment:
```suggestion
		// Format: `MediaBrowser key="value", key1="value1"`
```

Reply:
I swear I used key1=value1 key2=value2 at some point, no idea what happened to it.